### ISSUE TITLE
Source code changes?

### ISSUE THREAD

**Issue by papadeltasierra** [mmoskal/blackmagic-bluepill#2]
Hi mmoskal,
Really a query not an issue.  what source code changes did you have to make to map the pins?  Actually what I'm trying to do is figure out the pins of a standard image on a blue pill board but I'm finding the source code surprisingly hard to follow.
I'm hoping that your changes will highlight which bit of code I should be looking at!
Or alternatively, what are the standard pin-outs if you already know them?  Especially interested in the "target Vref" pin as many users seem to ignore this.
Thanks for any help you can offer,
Papadeltasierra (Paul DS.)

**Comment by vatrat** [comment]
Same issue!
